math/unsafe/rand.go

```diff
@@ -26,7 +26,6 @@ var grand *Rand
 
 func init() {
 	grand = NewRand()
-	grand.init()
```

Comment:
Can we can a changelog under math/CHANGELOG for this please.

Reply:
+1